project2/styles.css

```
@@ -0,0 +1,124 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(100, 100, 100);
+    color: rgb(255, 255, 255);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+}
+
+a {
+    text-decoration: none;
+    color: rgb(255, 255, 255);
+}
+
+h2 {
+    font-family: 'Catamaran', sans-serif;
+    font-size: 30px;
+    font-weight: bold;
```

Since you pulled in a specific weight of Catamaran, consider using that here:
```suggestion
    font-weight: 900;
```